BenadrylAutomation/src/main/java/com/jnj/pageobjects/ProductPage.java

```diff
@@ -0,0 +1,313 @@
+/**
+ * @author Zahin Ahad
+ * @date 4-March-24
+ */
+
+package com.jnj.pageobjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+
+import com.jnj.actions.Action;
+import com.jnj.base.BaseClass;
+
+public class ProductPage extends BaseClass{
+    Actions actions;
+
+    /**
+     * Constructor
+     */
+    public ProductPage() {
+        PageFactory.initElements(driver, this);
+        actions = new Actions(driver);
+    }
+
+    /**
+     * Locators
+     */
+    @FindBy(css = "#tab-menu-8651")
+    WebElement prodHeaderMenu;
+
+    @FindBy(css = "[itemprop='name'] > #node-title-url-386")
+    WebElement prodTileTitle;
+
+    @FindBy(css = "[alt='BENADRYL® Allergy ULTRATABS® Tablets']")
+    WebElement prodImg;
+
+    @FindBy(css = "#content-main")
+    WebElement prodTitle;
+
+    @FindBy(css = ".bv_ratings_summary")
+    WebElement prodRating;
+
+    @FindBy(css = ".inside > .pane-node-body")
+    WebElement prodOverview;
+
+    @FindBy(css = ".bv_war_button")
+    WebElement writeAReviewBtn;
+
+    @FindBy(css = ".zZXQY")
+    WebElement writeAReviewModal;
+
+    @FindBy(css = "#mini-panel-product_header .ps-button-label")
+    WebElement buyNowBtn;
+
+    @FindBy(css = ".ps-wtb")
+    WebElement buyNowModal;
+
+    @FindBy(css = ".jump-to-title")
+    WebElement jumpToTitle;
+
+    @FindBy(css = "[href=\"#product-overview\"] > h2")
+    WebElement prodOverviewHeading;
+
+    @FindBy(css = ".pane-node-field-product-directions")
+    WebElement directionSection;
+
+    @FindBy(css = ".pane-node-field-product-directions h2")
+    WebElement directionHeading;
+
+    @FindBy(id = "mini-panel-product_ingredients_modal")
+    WebElement ingredientsSection;
+
+    @FindBy(css = "#mini-panel-product_ingredients_modal h2")
+    WebElement ingredientsHeading;
+
+    @FindBy(css = "li #used-for")
+    WebElement usedForSection;
+
+    @FindBy(id = "tab_used-for")
+    WebElement usedForHeading;
+
+    @FindBy(css = "li #warnings")
+    WebElement warningsSection;
+
+    @FindBy(id = "tab_warnings")
+    WebElement warningsHeading;
+
+    @FindBy(id = "mini-panel-product_faq")
+    WebElement faqsSection;
+
+    @FindBy(css = "#mini-panel-product_faq span")
+    WebElement faqsHeading;
+
+    @FindBy(css = "#bv-product-reviews")
+    WebElement reviewsSection;
+	
+	@FindBy(xpath = "//div[@id='onetrust-close-btn-container']")
+	WebElement closePrivacyBtn;
+    
+    public WebElement getJumpToHeadings(String headingText) {
+        return driver.findElement(By.xpath("//div[@class='jump-to-wrapper']//ul//li//a[text()='" + headingText + "']"));
+    }
+	
+	/**
+	 * Function to click on close privacy pop-up button
+	 */
+	public void closePrivacyPopup() {
```

Comment:
@zahinSJI why this function is here? it is already there in the signup page.

Reply:
Function removed from POM file for PDP